Documentation/deploy-workers.md

```diff
@@ -45,11 +44,11 @@ After=flanneld.service
 
 #### Create the kubelet Unit
 
-Create `/etc/systemd/system/kubelet.service` and replace: 
+Create `/etc/systemd/system/kubelet.service` and replace the following variables: 
 
-* Replace `${MASTER_IP}`
-* Replace `${ADVERTISE_IP}` with this nodes publicly routable IP.
-* Replace `${DNS_SERVICE_IP}`
+* `${MASTER_IP}`
+* `${ADVERTISE_IP}`, with this node's publicly routable IP.
+* `${DNS_SERVICE_IP}`
```

Comment:
gah, I hate the "replace replace replace" redundancy. Let me have another swing.